HardCodedStringCheckerSharp/Properties/AssemblyInfo.cs

```diff
@@ -32,5 +32,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion( "1.4.0.0" )]
-[assembly: AssemblyFileVersion( "1.4.0.0" )]
+[assembly: AssemblyVersion( "1.5.0.0" )]
+[assembly: AssemblyFileVersion( "1.5.0.0" )]
```

Comment:
Good question!  I am thinking NuGet packages where it would.  So chocolatey?  I need to look.  Should this be a pre-build step with this as a NuGet package it would get seen.  Not sure if that helps any! :smiley: